README.md

```diff
@@ -24,28 +24,28 @@ The engine is used by several commercial game studios and computer-science cours
  - [Leap](https://gamejolt.com/games/leap/313308)
  - [Jumping Jack Flag](http://timealias.bplaced.net/jack/)
  - [PapaSpace Flight Simulation](https://www.papaspace.at/)
- - [Cubic Nightmare](https://jaredbgreat.itch.io/cubic-nightmare)
+ - [Cubic Nightmare (on Itch)](https://jaredbgreat.itch.io/cubic-nightmare)
  - [Chatter Games](https://chatter-games.com)
  - [Exotic Matter](https://exoticmatter.io)
  - [Demon Lord (on Google Play)](https://play.google.com/store/apps/details?id=com.dreiInitiative.demonLord&pli=1)
  - [Marvelous Marbles (on Steam)](https://store.steampowered.com/app/2244540/Marvelous_Marbles/)
  - [Boxer (on Google Play)](https://play.google.com/store/apps/details?id=com.tharg.boxer)
  - [Depthris (on Itch)](https://codewalker.itch.io/depthris)
+ - [Stranded (on Itch)](https://tgiant.itch.io/stranded)
 
 ## Getting started
 
 Go to https://github.com/jMonkeyEngine/sdk/releases to download the jMonkeyEngine SDK.
-[Read the wiki](https://jmonkeyengine.github.io/wiki) for a complete install guide. Power up with some SDK Plugins and AssetPacks and you are off to the races. At this point you're gonna want to [join the forum](http://hub.jmonkeyengine.org/) so our tribe can grow stronger.
+[Read the wiki](https://jmonkeyengine.github.io/wiki) for a complete install guide. You'll want to [join the forum](http://hub.jmonkeyengine.org/) so our tribe can grow stronger.
```

Comment:
Or to incorporate the original sentiment a little:
"You may also want to join the forum to get your questions answered, share your JME projects, or participate in the thriving JME community."  (thriving optional)  (I personally prefer 'forum' to mentions of a specific tech stack since this the third forum software setup since I've been a participant.)

Reply:
I've submitted a new wording that encourages and motivates forum membership without mentioning a tribe or specifying the forum's underlying technology. Please let me know if this adequately addresses your comments @codex128 and @pspeed42 .

Reply:
Looks good to me.